.github/workflows/deploy.yml

```diff
@@ -0,0 +1,18 @@
+
+name: Build
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
I think this needs to be changed to something like

```yaml
on:
  push:
    branches:
      - master
```

There's no "merged PR" condition, but I believe this will cover both merged PRs, and direct pushes to master (which we don't normally do but might be useful in an emergency).

Reply:
This already existed below so I just removed the PR condition in c4caf45

Reply:
Ah I didn't even notice. Cool.